test/inductor/test_b2b_gemm.py

```diff
@@ -0,0 +1,42 @@
+# Owner(s): ["module: inductor"]
+import torch
+from torch._inductor.test_case import run_tests, TestCase
+from torch._inductor.utils import run_and_get_code
+from torch.testing._internal.inductor_utils import HAS_CUDA
+
+
+class B2BGEMMTest(TestCase):
+    @torch._inductor.config.patch(b2b_gemm_pass=True)
+    def test_b2b_gemm_supported_block_size(self):
+        """sizes match the supported block sizes"""
+
+        def f(m1: torch.Tensor, m2: torch.Tensor, m3: torch.Tensor) -> torch.Tensor:
+            return torch.matmul(torch.matmul(m1, m2), m3)
+
+        f_opt = torch.compile(f)
+        A = torch.randn((512, 64), device="cuda", dtype=torch.float16)
+        B = torch.randn((64, 256), device="cuda", dtype=torch.float16)
+        C = torch.randn((256, 64), device="cuda", dtype=torch.float16)
```

Comment:
Is it possible to test against different shapes now? Do you think there would be any value in doing so?

Reply:
Currently it supports all shapes divisible by the block size; the remaining thing to implement is padding the non-divisible case (which I think should be easy to implement). I guess after that I can basically test any size here.

Reply:
Now can handle arbitrary shapes.